helix-view/src/editor.rs

```diff
@@ -21,10 +21,12 @@ use helix_core::Position;
 use serde::Deserialize;
 
 #[derive(Debug, Clone, PartialEq, Deserialize)]
-#[serde(rename_all = "kebab-case")]
+#[serde(rename_all = "kebab-case", default)]
 pub struct Config {
     /// Padding to keep between the edge of the screen and the cursor when scrolling. Defaults to 5.
     pub scrolloff: usize,
+    /// Number of lines to scroll at once. Defaults to 3
+    pub scroll_lines: isize,
```

Comment:
I changed it to `isize` for reversed scrolling and it makes sense in the code. This can be useful for some touchpad users who want natural scrolling (hold and drag contents, not view). Can be removed if considered unneeded.